doc/source/whatsnew/v1.2.5.rst

```diff
@@ -21,6 +21,13 @@ Fixed regressions
 
 .. ---------------------------------------------------------------------------
 
+.. _whatsnew_125.deprecations:
+
+Deprecations
+~~~~~~~~~~~~
+
+- Deprecated passing lists as ``key`` to :meth:`DataFrame.xs` and :meth:`Series.xs` (:issue:`41760`)
+
```

Comment:
I don't think we can backport this since it violates our policy.

```
pandas will introduce deprecations in **minor** releases. These deprecations
will preserve the existing behavior while emitting a warning that provide
guidance on:

* How to achieve similar behavior if an alternative is available
* The pandas version in which the deprecation will be enforced.

We will not introduce new deprecations in patch releases.
```

Reply:
Will make a pr later to move back to 1.3

Reply:
Thanks. just need to move the release note on master. I'll close the backport PR.